lib/Backend/Func.h

```diff
@@ -1100,6 +1099,22 @@ static const unsigned __int64 c_debugFillPattern8 = 0xcececececececece;
 public:
     Lowerer* m_lowerer;
 #endif
+
+    // Lazy bailout
+private:
+    bool                hasLazyBailOut : 1;
+    IR::LabelInstr*     lazyBailOutThunkLabel;
```

Comment:
>lazyBailOutThunkLabel [](start = 24, length = 21)

where do you use this field?

Reply:
It's not used for now. I put it there in case I need it in the future. Should I remove it?

Reply:
Yes.

---
In reply to: [241856350](https://github.com/Microsoft/ChakraCore/pull/5861#discussion_r241856350) [](ancestors = 241856350)

Reply:
Removed!